TechSupport-Session-Manager/src/main/java/com/github/RevatureTechSupport/TechSupportSessionManager/TechSupportSessionManagerApplication.java

```diff
@@ -6,8 +6,8 @@
 @SpringBootApplication
 public class TechSupportSessionManagerApplication {
 
-	public static void main(String[] args) {
-		SpringApplication.run(TechSupportSessionManagerApplication.class, args);
-	}
+    public static void main(String[] args) {
+        SpringApplication.run(TechSupportSessionManagerApplication.class, args);
+    }
 
```

Comment:
```suggestion
```

Reply:
this seems to have added in the indicators of a merge conflict, just need to remove these <<<<< ===== >>>>>